networkx/algorithms/clique.py

```diff
@@ -675,92 +571,6 @@ def node_clique_number(G, nodes=None, cliques=None, separate_nodes=False):
     return {n: size_for_n[n] for n in nodes}
 
 
-def number_of_cliques(G, nodes=None, cliques=None):
-    """Returns the number of maximal cliques for each node.
-
-    .. deprecated:: 3.0
-
-       number_of_cliques is deprecated and will be removed in v3.2.
-       Use the result of `find_cliques` directly to compute the number of
-       cliques containing each node::
-
-           {n: sum(1 for c in nx.find_cliques(G) if n in c) for n in G}
```

Comment:
Yeah, I like this function. Can we please keep it?

This isn't really a one liner for someone who doesn't know cliques and networkx API that well.

I don't mind adding this to `__getattr__` to give an explicit error message with the solution, but that also means we need to maintain text of the error message and IMO it's easier to test explicit functions than strings of error messages.

Reply:
Done! `number_of_cliques` has been added back in 5643f0b

Reply:
Thanks, shouldn't we also change the code to the one liner then?

Reply:
Well the `number_of_cliques` function has boilerplate for dealing with the `nodes` and cliques` kwargs that are required to support the current API.